net/gce.sh

```diff
@@ -272,6 +258,16 @@ while getopts "h?p:Pn:c:r:z:gG:a:d:uxf" opt "${shortArgs[@]}"; do
   esac
 done
 
+if [[ -n $customMachineType ]] ; then
+  bootstrapLeaderMachineType="$customMachineType"
+elif [[ -n $enableGpu ]] ; then
```

Comment:
I wonder if these should be checked independently and error out if they're both set, since they're mutually exclusive?  As is passing both will silently take `customMachineType` which may be unexpected behavior

Reply:
They are not mutually exclusive.  Passing `customMachineType` and `enableGpu` is valid, as `customMachineType` does not assume GPU or not, but relies on the presence/absence of the `enableGpu`

Reply:
Ah gotcha!  All good then